### ISSUE TITLE
Simple FileEditor cannot be launched in Wx

### ISSUE THREAD

**Issue by kitchoi** [enthought/traitsui#889]
To reproduce:
```
from traits.api import *
class FileModel(HasTraits):
    filepath = File()

m = FileModel()
m.configure_traits()
```

Output:
```
15:44:18: Debug: Adding duplicate image handler for 'Windows bitmap file'
15:44:18: Debug: Adding duplicate animation handler for '1' type
15:44:18: Debug: Adding duplicate animation handler for '2' type
15:44:18: Debug: Adding duplicate image handler for 'Windows bitmap file'
15:44:18: Debug: Adding duplicate animation handler for '1' type
15:44:18: Debug: Adding duplicate animation handler for '2' type
15:44:18: Debug: Adding duplicate image handler for 'Windows bitmap file'
15:44:18: Debug: Adding duplicate animation handler for '1' type
15:44:18: Debug: Adding duplicate animation handler for '2' type
Traceback (most recent call last):
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/view_application.py", line 152, in OnInit
    args=self.args,
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/view.py", line 462, in ui
    ui.ui(parent, kind)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/ui.py", line 246, in ui
    self.rebuild(self, parent)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/toolkit.py", line 125, in ui_live
    ui_live.ui_live(ui, parent)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_live.py", line 52, in ui_live
    _ui_dialog(ui, parent, BaseDialog.NONMODAL)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_live.py", line 88, in _ui_dialog
    BaseDialog.display_ui(ui, parent, style)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_base.py", line 71, in display_ui
    ui.owner.init(ui, parent, style)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_live.py", line 203, in init
    sw = panel(ui, window)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_panel.py", line 272, in panel
    panel, content[0], ui
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_panel.py", line 431, in fill_panel_for_group
    panel, group, ui, suppress_label, is_dock_window, create_panel
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_panel.py", line 587, in __init__
    self.add_items(content, panel, self.sizer)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/ui_panel.py", line 908, in add_items
    editor.prepare(item_panel)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/editor.py", line 264, in prepare
    self.init(parent)
  File "/Users/kchoi/Work/ETS/traitsui/traitsui/wx/file_editor.py", line 96, in init
    sizer.Add(control, 1, wx.EXPAND | wx.ALIGN_CENTER)
wx._core.wxAssertionError: C++ assertion "!(flags & (wxALIGN_BOTTOM | wxALIGN_CENTRE_VERTICAL))" failed at /Users/robind/projects/bb2/dist-osx-py36/build/ext/wxWidgets/src/common/sizer.cpp(2114) in DoInsert(): Vertical alignment flags are ignored with wxEXPAND
OnInit returned false, exiting...
```

The error looks similar to that in #828

**Comment by ievacerny** [comment]
Same applies to `DirectoryEditor`

**Comment by ievacerny** [comment]
Don't think it's worth opening another issue, so just commenting here.

A similar combination of alignment flags being ignored with `wxEXPAND` is raised when creating a toolbar for `TableEditor` with wx 4.1 (demos `examples/demo/Advanced/List_editors_demo.py` and `examples/demo/Standard_Editors/TableEditor_demo.py
`)
```
Traceback (most recent call last):

  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/view_application.py", line 152, in OnInit
    args=self.args,
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/view.py", line 462, in ui
    ui.ui(parent, kind)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/ui.py", line 246, in ui
    self.rebuild(self, parent)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/toolkit.py", line 125, in ui_live
    ui_live.ui_live(ui, parent)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_live.py", line 52, in ui_live
    _ui_dialog(ui, parent, BaseDialog.NONMODAL)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_live.py", line 88, in _ui_dialog
    BaseDialog.display_ui(ui, parent, style)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_base.py", line 71, in display_ui
    ui.owner.init(ui, parent, style)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_live.py", line 203, in init
    sw = panel(ui, window)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_panel.py", line 272, in panel
    panel, content[0], ui
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_panel.py", line 431, in fill_panel_for_group
    panel, group, ui, suppress_label, is_dock_window, create_panel
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_panel.py", line 571, in __init__
    content, ui, panel, group, self.add_notebook_item
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_panel.py", line 364, in create_notebook_for_it
ems
    item_handler(item, panel, sizer)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_panel.py", line 698, in add_notebook_item
    self.add_items([item], parent, sizer)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/ui_panel.py", line 908, in add_items
    editor.prepare(item_panel)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/editor.py", line 264, in prepare
    self.init(parent)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/table_editor.py", line 200, in init
    self._create_toolbar(panel, sizer)
  File "/Users/icernyte/Documents/repos/traitsui/traitsui/wx/table_editor.py", line 456, in _create_toolbar
    sizer.Add(tb_sizer, 0, wx.ALIGN_RIGHT | wx.EXPAND)
wx._core.wxAssertionError: C++ assertion "!(flags & (wxALIGN_RIGHT | wxALIGN_CENTRE_HORIZONTAL))" failed at /U
sers/robind/projects/bb2/dist-osx-py36/build/ext/wxWidgets/src/common/sizer.cpp(2089) in DoInsert(): Horizonta
l alignment flags are ignored with wxEXPAND
```

**Comment by kitchoi** [comment]
The original issue for FileEditor is solved via #1557
The issue about TableEditor was ticketed separately in #1094, and solved via #1095
Closing.
